src/test/daml/Daml/Finance/Instrument/Equity/Test/Dividend.daml

```diff
@@ -148,3 +148,110 @@ run = script do
     (investor, 2000.50, investorCashHoldingCid)]
 
   pure ()
+
+-- Bonus issue (additional shares): lifecycling and settlement
```

Comment:
I wonder if it makes sense to embed this in the previous test.

That would reduced the amount of boiler-place , wdyt?

Reply:
You mean adding the new parts after the existing test in `run`, as a new test within the same function?